src/user.rs

```diff
@@ -439,6 +478,19 @@ async fn pick_random_room(rooms: &RwLock<Vec<OwnedRoomId>>) -> Option<OwnedRoomI
         .map(|room| room.to_owned())
 }
 
+async fn pick_random_channels(channels: &RwLock<HashSet<OwnedRoomId>>) -> Option<OwnedRoomId> {
+    let channels = channels.read().await;
+    if !channels.is_empty() {
```

Comment:
No need for this check, `choose` will automatically return `None` if the `slice/Vec` is empty